src/util/EventLogger.js

```diff
@@ -96,15 +101,21 @@ export function logPurchaseFlowEvent(
     }
     if (window.doNotTrack || navigator.doNotTrack) return;
     if (vue.$gtag) {
+      let actualPrice = price;
+      let actualCurrency = currency;
+      if (actualPrice && actualCurrency === 'LIKE') {
+        actualPrice *= ROUGH_LIKE_TO_USD_PRICE;
+        actualCurrency = 'USD';
+      }
       vue.$gtag.event(event, {
         transaction_id: txHash,
-        value: price,
+        value: actualPrice,
         items: [
           {
             item_id: classId,
             item_name: name,
-            currency,
-            price,
+            currency: actualCurrency,
+            price: actualPrice,
```

Comment:
I think should use this computed prop
https://github.com/likecoin/liker-land/blob/64acab5bd2a51d9dac4222ff7720ad535e08610b/src/mixins/nft.js#L146-L148

for this
https://github.com/likecoin/liker-land/blob/64acab5bd2a51d9dac4222ff7720ad535e08610b/src/mixins/nft.js#L397

Reply:
Rewrote the last commit and force pushed